client/src/components/games/AvailableGames.tsx

```diff
@@ -178,7 +178,7 @@ function AvailableGames() {
             </NavLink>
           </div>
         ))}
-        {venueFilter === '' || (Array.isArray(filteredGames) && filteredGames.length === 0) &&
+        {(Array.isArray(filteredGames) && filteredGames.length === 0) &&

```

Comment:
the initial value of filtereGames is [], so no need to check whether its array or not
```suggestion
{filteredGames.length === 0 && filteredGames.map(game => (
```